spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/SparkTemplate.scala

```diff
@@ -27,6 +27,7 @@ object SparkTemplate {
   val defaultConf = new SparkConf(true)
     .set("spark.cassandra.connection.host", getHost(0).getHostAddress)
     .set("spark.cassandra.connection.port", getPort(0).toString)
+    .set("spark.ui.enabled", "false")
```

Comment:
Sometimes I like having this on while IT is running, is it important for testing to disable it? I'm ok if it makes test more stable in general